src/app/calendar/page.tsx

```diff
@@ -4,18 +4,54 @@ import Link from "next/link";
 export const revalidate = 3600;
 
 const scheduleData = [
+  {
+    title: "DATA Book Club & Mappy Hour",
+    date: new Date("2025-02-22 16:00-05:00"),
+    points: [
+      "Sunday, February 23 from 1:00PM - 4:00PM",
+      "Dolce Paradiso in Addison (5030 Addison Cir, Addison)",
+      "Nearest station: Addison Transit Center",
+      "Reading Human Transit for the month of February",
+      "Stick around afterwards for Mappy Hour, an informal social for local urbanists to contribute to OpenStreetMap together.",
+    ],
+  },
   {
     title: "DATA February General Meeting",
     date: new Date("2025-02-22 16:00-05:00"),
     points: [
       "Saturday, February 22 from 1:00PM - 4:00PM",
```

Comment:
time should be modified to 1:30pm

Reply:
Also update Line 76 in remembering-mary.md for the same reason

Reply:
Done